v0/sources/shopify/transform.js

```diff
@@ -154,6 +154,8 @@ const processEvent = inputEvent => {
     name: "RudderStack Shopify Cloud",
     version: "1.0.0"
   });
+  // adding the subscribed topic inside context.topic
```

Comment:
I think you can remove this comment, line 158 is explanatory enough. If a comment is needed to solve any possible confusion specific to this change then you can add a comment explaining the need for this change.